NativeAuthSampleApp/MultiFactorAuthenticationViewController.swift

```diff
@@ -0,0 +1,329 @@
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import MSAL
+import UIKit
+
+/**
+ * MultiFactorAuthenticationViewController class implements samples for using Multi-Factor Authentication (MFA) via Email OTP.
+ * The code shows how to sign in a user with password (1st factor), and how to request and submit an email OTP code (2nd factor).
```

Comment:
```suggestion
 * The class shows how to sign in a user with password (1st factor), and how to request and submit an email OTP code (2nd factor).
```
better to use "class" isntead of "code" because the latter is also used as email OTP code

Reply:
Yes, I agree. Thanks for the catch.